docs/PacketFence_Clustering_Guide.asciidoc

```diff
@@ -645,6 +645,289 @@ As the daily automatic backups, you will find the files in:
+
+ [DC2 pf2-dc2.example.com]
+ management_ip=192.168.2.12
+ 
+ [DC2 pf2-dc2.example.com interface ens192]
+ ip=192.168.2.12
+----
+
+Notes on the configuration:
+
+ * The hostnames (pf1-dc1.example.com, pf2-dc1.example.com, etc) are not directly related to the cluster logic and the servers can have any hostname without impacting the cluster behavior. The assignment of a server to a cluster zone is made by the first part of the section name (ex: "DC1 pf.example.com" assigns server "pf.example.com" to cluster zone "DC1")
+ * Each cluster zone needs to have its own "CLUSTER" definition that declares the virtual IPs to use for this cluster zone. This also declares the management IP on which the zone should be joined.
+ * Given your zones aren't in the same layer 2 network, you cannot use the same virtual IP between your zones.
+ * You should always declare a "CLUSTER" definition even though a zone has only a single server.
+ * Your network equipment should point RADIUS authentication and accounting to both virtual IPs (192.168.1.10 and 192.168.2.10 in this example) either in primary/secondary or load-balancing mode.
+ * RFC3576 servers (CoA and RADIUS disconnect) should be declared on your network equipment (if supported) for both virtual IPs (192.168.1.10 and 192.168.2.10 in this example)
+
+RADIUS server with captive-portal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+WARNING: As shortly explained above, deploying a captive-portal across a layer 3 network using the cluster zones is complex and requires in-depth knowledge of networking and how PacketFence works.
+
+In order to configure a RADIUS server with a captive-portal on a layer 3 cluster, you will need at least 3 servers (5 are used in this example) with 2 interfaces (one for management and one for registration)
+
+NOTE: Isolation is omitted in this example for brievety and should be configured the same way as registration if needed
+
+Cluster design and behavior:
+
+ * This example will use 3 servers in a network (called DC1), and 2 in another network (called DC2).
+ * Each group of server (in the same L2 network) will have a virtual IP address and will perform load-balancing (RADIUS, HTTP) to members in the same L2 zone (i.e. same network).
+ * All the servers will use MariaDB Galera cluster and will be part of the same database cluster meaning all servers will have the same data.
+ * In the event of the loss of DC1 or a network split between DC1 and DC2, the databases on DC2 will go in read-only and will exhibit the behavior described in "Quorum behavior".
+ * All the servers will share the same configuration and same cluster.conf. The data in cluster.conf will serve as an overlay to the data in pf.conf and networks.conf to perform changes specific to each layer 3 zone.
+
+The schema below presents the routing that needs to be setup in your network in order to deploy this example:
+
+image::cluster-zones-registration.png[scaledwidth="100%",alt="Cluster zones with registration network"]
+
```

Comment:
The schema is a little bit confusing for me and maybe we will need to add another network for routing.
Per example use 192.168.33.0/30 (to reach DC1) and 192.168.33.4/30 (to reach DC2).